python_filmaffinity/client.py

```diff
@@ -73,7 +75,7 @@ def _get_movie_by_args(self, key, value, trailer=False):
         if key in FIELDS_MOVIE:
             options = '&stype[]=%s' % key
             url = self.url + 'advsearch.php?stext=' + \
-                str(value) + options
+                  str(value) + options
```

Comment:
pep8 python_filmaffinity

python_filmaffinity/client.py:78:19: E127 continuation line over-indented for visual indent